src/Controller/AbsenceController.php

```diff
@@ -15,59 +16,271 @@ class AbsenceController extends BaseController
+                $etatStyle = $elem['valide'] == 0 ? "font-weight:bold;" : null;
+                $etatStyle = $elem['valide'] < 0 ? "color:red;" : $etatStyle;
+
+                $pj1Checked = $elem['pj1'] ? "checked='checked'" : null;
+                $pj2Checked = $elem['pj2'] ? "checked='checked'" : null;
+                $soChecked = $elem['so'] ? "checked='checked'" : null;
+
+                $begin = dateFr($elem['debut'], true);
+                $end = dateFr($elem['fin'], true);
+
+                $agentsList = implode($elem['agents'], ",");
+                $rrule = $elem['rrule']?$elem['rrule']:null;
+                $commentaires = $elem['commentaires'];
+                $motif = $elem['motif'];
+                $requestDate = dateFr($elem['demande'], true);
+
+                foreach($absdocs as $absdoc){
+                   $absLinks[]=array(
+                    'link' =>"absences/document/". $absdoc->id() . "target='_blank'",
+                    'name' => $absdoc->filename()
+                  );
+                }
+
+                $absList[] = array(
+                  'id'            => $id,
+                  'rrule'         => $rrule,
+                  'agentsList'    => $agentsList,
+                  'state'         => $etat,
+                  'stateStyle'    => $etatStyle,
+                  'absLinks'      => $absLinks,
+                  'pj1Checked'    => $pj1Checked,
+                  'pj2Checked'    => $pj2Checked,
+                  'soChecked'     => $soChecked,
+                  'motive'        => $motif,
+                  'comments'      => $commentaires,
+                  'requestDate'   => $requestDate,
+                  'begin'         => $begin,
+                  'end'           => $end
+                );
+
+            }
         }
 
+
+        $this->templateParams(array(
+            'admin'                 => $admin,
+            'begin'                 => dateFr($debutSQL),
+            'end'                   => dateFr($finSQL),
+            'sort'                  => $sort,
+            'perso_id'              => $perso_id,
+            'agentsMenu'            => $agents_menu,
+            'right6'                => in_array(6, $this->droits)?1:0,
+            'right701'              => in_array(701, $this->droits)?1:0,
+            'deletedAgents'         => $agents_supprimes ,
+            'selectedAgents'        => $selectedAgents ,
+            'absences'              => $absList,
+            'absences_validation'   => $this->config('Absences-validation')
+        ));
+
+        return $this->output('absences/index.html.twig');
+    }
+
+    /**
+     * @Route("/absence/add", name="absence.add", methods={"GET"})
+     */
+    public function add(Request $request)
+    {
+        $confirm = $request->get('confirm');
+        $id = $request->get('id');
+
+        $this->dbprefix = $GLOBALS['dbprefix'];
```

Comment:
This variable is not needed now

Reply:
Not fixed

Reply:
Still not fixed

Reply:
absenceInfos() required this var, as the function is called through setCommonTemplateParams